Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,138 @@
 import React from 'react';
 import './App.css';
+import ActionButton from './components/ButtonComponents/ActionButton.js';
+import NumberButton from './components/ButtonComponents/NumberButton.js';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay.js';
 
+const nums = [1,2,3,4,5,6,7,8,9];
+let total = 0;
+let formula = "";
+let displayFormula = "";
+const NumberPanel = props =>{
+    return props.function;
+  }
+
+/*
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className = "calculator">
+        <CalculatorDisplay content = {total}/>
+        <div className = "button-groups">
+            <div className = "left-button-group">
+              <ActionButton content = "clear" style ="action-button-top" function = {() =>updateTotal("clear")}/>
+              <NumberPanel />
+              <ActionButton content = "0" style = "action-button-bottom" function = {() =>updateTotal("0")}/>
+            </div>
+
+          <div className = "right-button-group">
+              <NumberButton buttonStyle = "function-button" content = {"\u00f7"} function = {()=>updateTotal("\u00f7")}/>
+              <NumberButton buttonStyle = "function-button" content = {"\u00d7"} function = {()=>updateTotal("\u00d7")}/>
+              <NumberButton buttonStyle = "function-button" content = {"-"} function = {()=>updateTotal("-")} />
+              <NumberButton buttonStyle = "function-button" content = {"+"} function = {()=>updateTotal("+")} />
+              <NumberButton buttonStyle = "function-button" content = {"="} function = {()=>updateTotal("=")} />
+
+
+
+
+
+          </div>
+        </div>
     </div>
   );
 };
+*/
+
+class App extends React.Component{
+  constructor(){
+    super();
+    this.state = {
+      total: 0,
+      display: 0
+    };
+  }
+
+   updateTotal = (input) =>{
+    console.log(input);
+      if(input==="="){
+        total = eval(formula);
+        console.log(total);
+        this.setState({total: total});
+        this.setState({display:total});
+
+        formula = "";
+        displayFormula="";
+        return;
+
+      }
+      if(input ==="\u00d7"){
```

Comment:
I see that you have some previous experience with c#. It's good to bring that in for use.